src/components/TagInput/themes/dark/index.js

```diff
@@ -0,0 +1,180 @@
+import {
```

Comment:
Is `TagInput` and the other files part of some library? It appears copied from somewhere. Does this need to live in our code?